tasks/src/unconditional-image-generation/about.md

```diff
@@ -0,0 +1,22 @@
+## Use cases
+In the cartoon anime industry, it is normally time-consuming to draw images of characters, objects 
+as well as landscapes because there is a huge demand on images and every little detail of each image
+needs to be taken into account. Furthermore, hiring high qualified artists cannot always be possible. 
+In this case, unconditional image generation models can be used to generate many anime images economically.
+
+Choosing a hairstyle or makeup that fit one's face is not always easy without trying them before. 
```

Comment:
Are you sure? As said, unconditional image generation is usually used to generate image from simple noise (and the noise is generated from a seed) you don't really input anything like you'd do with text-to-image models.

Reply:
thank you for your comments i will do more research and let you know the questions i may have